phpBB/styles/prosilver/theme/bidi.css

```diff
@@ -596,7 +604,11 @@ li.breadcrumbs span:first-child > a {
 */
 
 .rtl .caret {
-	float: right;
+	left: -6px;
+	right: inherit;
+	border-right: 1px solid;
+	border-right-color: inherit;
+	border-left: none;
 }
 
```

Comment:
there is an alternate change for this gets the same result
```css
.rtl .caret {
	border-right: 1px solid;
	border-right-color: inherit;
	border-left: none;
	right: 6px;
}
```
which is the best first or second?